src/clusterfuzz/_internal/issue_management/issue_tracker_policy.py

```diff
@@ -165,6 +172,11 @@ def _apply_new_issue_properties(self, policy, issue_type, is_crash):
       if non_crash_labels:
         policy.labels.extend(_to_str_list(non_crash_labels))
 
+    # Find all _ext_ keys and add to extension_fields
+    for k, v in self._data.items():
+      if k.startswith('_ext_'):
```

Comment:
nit: can you put '_ext_' into a class CONSTANT?

Reply:
Done.